_includes/themes/bootstrap/news/mica2-5.3-release.html

```diff
@@ -0,0 +1,32 @@
+<p xmlns="http://www.w3.org/1999/html">We are pleased to announce that Mica2 5.3 is now available. <a
+    href="/pages/products/mica/">Mica</a> is OBiBa’s data web portal for epidemiological studies and harmonization
+  initiatives.</p>
+
+<p class="obiba-top-margin">
+  This release includes the new feature related to harmonization protocols, namely, the retrieval of harmonization status statistics per study-specific datasets. Currently, only a handful of cohorts use this feature, here are some examples:
+</p>
+
+<ul class="obiba-no-style-list">
+  <li><i class="fa fa-caret-right obiba-small-right-margin"></i>
+    <a href="https://www.maelstrom-research.org/dataset/canpath-bl-hrfq-hp/"  target="_blank">Maelstrom Research <i class="fa fa-external-link-square"></i></a>
+  </li>
+  <li><i class="fa fa-caret-right obiba-small-right-margin"></i>
+    <a href="https://portal.canpath.ca/dataset/cptp_bl_hrfq"  target="_blank">The Canadian Partnership for Tomorrow's Health (CanPath) <i class="fa fa-external-link-square"></i></a>
+  </li>
+  <li><i class="fa fa-caret-right obiba-small-right-margin"></i>
+    <a href="https://portal-dev.citf.mcgill.ca/dataset/citf-bl-demographics-hp"  target="_blank">COVID-19 Immunity Task Force (CITF) <i class="fa fa-external-link-square"></i></a>
+  </li>
+</ul>  
+
+<p class="obiba-top-margin">See the percentages under each study-specific dataset in the harmonization potential table. A mouse over will provide the detailed counts of each harmonization status.</p>
+
+<div class="alert alert-info obiba-top-margin" role="alert">
+This feature is not in the current Mica public pages but will be included in the next release.
```

Comment:
Could you elaborate: it is a new feature but it is not included? I guess it is a backend feature that is not rendered in the frontend. Is there a procedure to get the visual?

Reply:
Updated the PR. Did not added tech details due to complexity.